src/health_manager.py

```diff
@@ -13,140 +14,179 @@
 from ui import view, meters
 from ui_manager import ScreenObjects, is_visible
 
-pause_state = True
+_pause_state = True
```

Comment:
given that we're exposing getters and setters, giving this variable an `_` prefix will indicate it's "private" status